finch@.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Finch daemon %I
+Documentation=https://runfinch.com https://builderhub.corp.amazon.com/docs/docker.html#finch https://github.com/runfinch/finch-daemon
+After=network.target local-fs.target
+
+[Service]
+ExecStart=/usr/local/bin/finch-daemon --socket-owner %i
+ExecStartPost=-rm -rf /var/run/docker.sock
+ExecStartPost=ln -s /run/finch.sock /var/run/docker.sock
```

Comment:
this i am not sure is a good idea by default, there might be case where customer wants to run both finch and docker, so any reference to docker doesnt seem correct.

Reply:
This is only inside the VM, which users should not be modifying